### PR TITLE
Update urls

### PR DESCRIPTION
**Pull request by ajduet**
I did a little more than just update the URLs here. I am attempting to remove the DB dependency as was discussed.

### CODE REVIEW

File: src/main/java/com/revature/sfadapter/services/SFAuthService.java [hunk 1]
```diff
@@ -1,86 +1,93 @@
...
-		return url;
+		return data;
 	}
 
-	public void sendForAuth(HttpServletResponse response, String clientRedirect) throws IOException{
-		String url = getRequestUrl(clientRedirect);
-		response.sendRedirect(url);
+	public String getAuthUrl(String clientRedirect) throws IOException{
+		LOGGER.debug("Building url for sales force request");
+		String authUrl = buildAuthorizationUrl("/services/oauth2/authorize", clientRedirect);
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/services/SFAuthService.java:65]: MAJOR Remove this unused "authUrl" local variable. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "authUrl". [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS1854)

File: src/main/java/com/revature/sfadapter/services/SFAuthService.java [hunk 2]
```diff
@@ -1,86 +1,93 @@
...
 import java.net.URLEncoder;
-import java.util.*;
+import java.util.Arrays;
+import java.util.Map;
 
 @Service
 public class SFAuthService {
 
-	private DaoService<SFWSAccessObject, String> tokenService;
+	private static final String clientId;
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/services/SFAuthService.java:24]: MINOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS00115)

File: src/main/java/com/revature/sfadapter/services/SFAuthService.java [hunk 3]
```diff
@@ -1,86 +1,93 @@
...
 
 	private static final Logger LOGGER = Logger.getLogger(SFAuthService.class);
-	@Autowired
-	public void setTokenService(DaoService<SFWSAccessObject, String> tokenService){
-		Assert.notNull(tokenService);
-		this.tokenService = tokenService;
+
+	private String buildAuthorizationUrl(String endpoint, String clientRedirect){
+
+		//StringBuilder url = new StringBuilder(String.format("%s?response_type=code", authorizationUrl));
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/services/SFAuthService.java:41]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/main/java/com/revature/sfadapter/services/SFAuthService.java [hunk 4]
```diff
@@ -1,86 +1,93 @@
...
 		HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<>(data, headers);
 		System.out.println(request.toString());
-		System.out.println("Post ready sending to Salesforce");
-		ResponseEntity<SFWSAccessObject> obj = client.postForEntity("https://test.salesforce.com/services/oauth2/token", request, SFWSAccessObject.class);
-		return obj.getBody();
 
-	}
+		LOGGER.debug("Request ready sending to Salesforce");
+		LOGGER.error("sendForAccess is incomplete");
+		ResponseEntity<SFWSAccessObject> obj = client.postForEntity(authorizationUrl, request, SFWSAccessObject.class);
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/services/SFAuthService.java:89]: MAJOR Remove this unused "obj" local variable. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "obj". [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS1854)

File: src/main/java/com/revature/sfadapter/services/SFAuthService.java [hunk 5]
```diff
@@ -1,86 +1,93 @@
...
-import java.util.*;
+import java.util.Arrays;
+import java.util.Map;
 
 @Service
 public class SFAuthService {
 
-	private DaoService<SFWSAccessObject, String> tokenService;
+	private static final String clientId;
+	private static final String clientSecret;
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/services/SFAuthService.java:25]: MINOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS00115)

File: src/main/java/com/revature/sfadapter/services/SFAuthService.java [hunk 6]
```diff
@@ -1,86 +1,93 @@
...
+	private static final String authorizationUrl;
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/services/SFAuthService.java:26]: MINOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS00115)

File: src/main/java/com/revature/sfadapter/services/SFAuthService.java [hunk 7]
```diff
@@ -1,86 +1,93 @@
...
+	private static final String myCallbackUrl;
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/services/SFAuthService.java:27]: MINOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AS00115)

File: src/main/java/com/revature/sfadapter/web/AuthController.java [hunk 8]
```diff
@@ -2,6 +2,7 @@
 
 import com.revature.sfadapter.services.SFAuthService;
 import com.revature.sfadapter.util.SFWSAccessObject;
```
Comment (ghost) [inline src/main/java/com/revature/sfadapter/web/AuthController.java:4]: MINOR Remove this unused import 'com.revature.sfadapter.util.SFWSAccessObject'. [rule](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by ghost** [comment]
SonarQube analysis reported 10 issues
* MAJOR 5 major
* MINOR 5 minor

Watch the comments in this conversation to review them.

**Review by tjkemper — APPROVED** [review]
lgtm

**Pull request merged**